src/test/java/com/squareup/kotlinpoet/FunSpecTest.kt

```diff
@@ -221,6 +221,22 @@ class FunSpecTest {
       |fun foo(string: kotlin.String, nodoc: kotlin.Boolean): kotlin.String = "foo"""".trimMargin())
   }
 
+  @Test fun functionWithModifiedReturnKdoc() {
+    val funSpec = FunSpec.builder("foo")
+        .addParameter(ParameterSpec.builder("nodoc", Boolean::class).build())
+        .returns(String::class, "the foo.")
+        .addCode(CodeBlock.of("return \"foo\""))
```

Comment:
supernit: `.addCode("return %S", "foo")` is the pattern we generally adhere to

Reply:
TIL. The one I did always felt awkward. Have fixed it in a test above this one too which had it.